pages/common/bat.md

```diff
@@ -4,26 +4,34 @@
 > A `cat` clone with syntax highlighting and Git integration.
 > More information: <https://github.com/sharkdp/bat>.
 
-- Print the contents of one or more files to `stdout`:
+- Pretty print the contents of one or more files to `stdout`:
 
 `bat {{path/to/file1 path/to/file2 ...}}`
 
 - Concatenate several files into the target file:
 
 `bat {{path/to/file1 path/to/file2 ...}} > {{path/to/target_file}}`
 
-- Append several files into the target file:
+- Remove decorations and auto paging:
 
-`bat {{path/to/file1 path/to/file2 ...}} >> {{path/to/target_file}}`
+`bat {{--style=plain --pager=never|-pp}} {{path/to/file}}`
```

Comment:
```suggestion
`bat --style=plain --pager={{never|-pp}} {{path/to/file}}`
```

Reply:
The fact that this suggestion was made means that a potential user may also understand `|-pp` as an argument to `--pager`. How about mentioning it in the description instead?